<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tech_course_view</class>
 <widget class="QWidget" name="tech_course_view">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>职工号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>课程号：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_course_id">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_tch_id">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbn_clear">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>330</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbn_add">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>330</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbn_del">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>330</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_q_cid">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>390</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>课程编号：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbn_query">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>440</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>230</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>开课日期：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_since">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>220</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbn_save">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>330</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>修改日期</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>400</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>职工号：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_q_tid">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>390</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
